instance/openstack.py

```diff
@@ -99,39 +103,88 @@ def get_server_public_address(server):
     return first_address
 
 
-def get_swift_connection(
+def swift_service(
```

Comment:
`python-switclient` contains two separate API's to interact with swift. One is request oriented (one call is one request) and other one is focused on batch operations. I needed to use the "batch" operation's one, so it made sense to rewrite existing calls to this new service.